misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueModule.kt

```diff
@@ -102,38 +111,84 @@ class AwsSqsJobQueueModule(
       config.task_queue ?: RepeatedTaskQueueConfig(num_parallel_tasks = -1)
 
   private class AmazonSQSProvider(
-    val config: AwsSqsJobQueueConfig, val region: AwsRegion, val forSqsReceiving: Boolean
+    val config: AwsSqsJobQueueConfig,
+    val region: AwsRegion,
+    val forSqsReceiving: Boolean
   ) : Provider<AmazonSQS> {
     @Inject lateinit var credentials: AWSCredentialsProvider
+    @Inject lateinit var features: FeatureFlags
+    @Inject lateinit var appName: String
```

Comment:
does this need to be qualified with an `@AppName` annotation?

Reply:
aw crap, you're absolutely right!

- [ ] annotate `appName` with `@AppName`

Reply:
Done; good catch!

![impossible catch](https://user-images.githubusercontent.com/337352/106688929-ead6d100-6583-11eb-90e1-f74ee4d2fe94.gif)